kolibri/content/management/commands/importcontent.py

```diff
@@ -23,6 +23,26 @@ def add_arguments(self, parser):
         # directory to be given. Another is the 'importcontent network'
         # command to be given, where we'll expect a channel.
 
+        # However, some optional arguments apply to both groups. Add them here!
+        node_ids_help_text = """
+        Specify one or more node IDs to import. Only the files associated to those node IDs will be imported.
+        Make sure to call this last in the argument list.
+
+        e.g.
+
+        kolibri manage importcontent network <channel id> --node_ids <id1> <id2> [<ids>...]
+        """
+        parser.add_argument(
+            "--node_ids", "-n",
+            # Split the comma separated string we get, into a list of strings
+            type=lambda x: x.split(","),
```

Comment:
Could you just use the `nargs="+"` keyword here to allow multiple values? https://docs.djangoproject.com/en/1.11/howto/custom-management-commands/#module-django.core.management